nipype/interfaces/fsl/utils.py

```diff
@@ -262,6 +262,70 @@ def _format_arg(self, name, trait_spec, value):
         return super(Smooth, self)._format_arg(name, trait_spec, value)
 
 
+class SliceInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, argstr="%s", position=0, mandatory=True,
+                   desc="input filename")
```

Comment:
Since this places outputs in the same directory as `in_file` (unless `out_base_name` is defined), you should add `copyfile=False` to this trait.